charts/registry/templates/_helpers.tpl

```diff
@@ -0,0 +1,48 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "chart.name" -}}
```

Comment:
To be honest, I'm not sure, if using `chart` is a good idea or not. Because if I remember correctly, all "define" methods are registered in the same context. This means that if someone includes the DTR helm chart in another helm chart as a dependency (as we plan to do with the umbrella helm chart), we might run into problems if another helm chart defines the same methods. But I'm really not sure about that.

@evegufy do you have more experience with this? Otherwise I would just do a quick test on my local computer and test that.

Reply:
@florianrusch-zf My guess is that it remains in the context of the subchart even if you add it as dependency to another chart. Still, it's good to test it on your local.

Reply:
@florianrusch-zf @tunacicek even tough it may not lead to immediate issues, I think it's a good practice to define all templates in a unique manner, 'dtr.name' for instance would be better. And by doing so, we may also save some troubleshooting time in the future for use case like an umbrella chart.

Reply:
@evegufy good idea. Thanks :) 
I adjust the templates.